build/scripts-3.4/pydeface.py

```diff
@@ -0,0 +1,180 @@
+#!/opt/local/bin/python
+"""
```

Comment:
My thinking is python2 is required, because installing traits with python3 just failed for me,